Description: Configure service_plugin by default
 service_plugins = neutron.services.l3_router.l3_router_plugin.L3RouterPlugin
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2013-12-05

--- neutron-2013.2.orig/etc/neutron.conf
+++ neutron-2013.2/etc/neutron.conf
@@ -48,7 +48,7 @@ lock_path = $state_path/lock
 core_plugin = neutron.plugins.openvswitch.ovs_neutron_plugin.OVSNeutronPluginV2
 
 # Advanced service modules
-# service_plugins =
+service_plugins = neutron.services.l3_router.l3_router_plugin.L3RouterPlugin
 
 # Paste configuration file
 # api_paste_config = api-paste.ini
